### ISSUE TITLE
As an editor I want to be able to leave the alt tag of an image empty for certain images

### ISSUE THREAD

**Issue by svsven** [equinor/energyvision#548]
Publishing team would like to have the alt tag on images not be required for all images.

Per feedback from SEO company it should be fine to have an empty alt tag for images that only serve as decoration (for example certain illustrations). Need to figure out what the impact would be/how to handle this in terms of accessibility however.
